catshadow: Investigate segmentation fault in processPANDAUpdate

From: https://github.com/katzenpost/katzenpost/actions/runs/3619808783/jobs/6128213965
End-to-end tests failed with the following 
```
19:00:28.332 DEBU catshadow: PANDA exchange completed
17834
panic: runtime error: invalid memory address or nil pointer dereference
17835
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x7a0aa2]
17836

17837
goroutine 2835 [running]:
17838
golang.org/x/crypto/curve25519.ScalarMult(0x7f69e3736fe0, 0x0, 0x1540960)
17839
    /go/pkg/mod/golang.org/x/crypto@v0.1.0/curve25519/curve25519.go:26 +0x62
17840
golang.org/x/crypto/curve25519.ScalarBaseMult(...)
17841
    /go/pkg/mod/golang.org/x/crypto@v0.1.0/curve25519/curve25519.go:81
17842
github.com/katzenpost/doubleratchet.(*Ratchet).completeKeyExchange(0xc001bbe840, 0xc000730ea0)
17843
    /go/pkg/mod/github.com/katzenpost/doubleratchet@v0.0.13/ratchet.go:459 +0x1ff
17844
github.com/katzenpost/doubleratchet.(*Ratchet).ProcessKeyExchange(0xc001177018?, {0xc0003240e0, 0xdd, 0xdd})
17845
    /go/pkg/mod/github.com/katzenpost/doubleratchet@v0.0.13/ratchet.go:442 +0x1b9
17846
github.com/katzenpost/katzenpost/catshadow.(*Client).processPANDAUpdate(0xc000e62200, 0xc0003e5d30)
17847
    /go/katzenpost/catshadow/client.go:981 +0x18ad
17848
github.com/katzenpost/katzenpost/catshadow.(*Client).worker(0xc000e62200)
17849
    /go/katzenpost/catshadow/worker.go:122 +0x12ed
17850
github.com/katzenpost/katzenpost/core/worker.(*Worker).Go.func1()
17851
    /go/katzenpost/core/worker/worker.go:39 +0x74
17852
created by github.com/katzenpost/katzenpost/core/worker.(*Worker).Go
17853
    /go/katzenpost/core/worker/worker.go:37 +0x117
17854
exit status 2
17855
FAIL    github.com/katzenpost/katzenpost/catshadow    599.792s
17856
Error: make: *** [Makefile:10: dockerdockertest] Error 1
17857
Error: Process completed with exit code 2.
```

--- a/go.mod
+++ b/go.mod
@@ -13,7 +13,7 @@ require (
 	github.com/jackc/pgx v3.6.2+incompatible
 	github.com/katzenpost/chacha20 v0.0.0-20190910113340-7ce890d6a556
 	github.com/katzenpost/chacha20poly1305 v0.0.0-20211026103954-7b6fb2fc0129
-	github.com/katzenpost/doubleratchet v0.0.14
+	github.com/katzenpost/doubleratchet v0.0.15
 	github.com/katzenpost/noise v0.0.3
 	github.com/katzenpost/nyquist v0.0.0-20220905145943-9f7e8b431eaf
 	github.com/prometheus/client_golang v1.14.0

